integration_tests/mockApis/responses/expectedArrivals.ts

```diff
@@ -1,4 +1,32 @@
 export default {
+  arrival: ({
+    fromLocationType,
+    isCurrentPrisoner,
+    dateOfBirth = '1970-01-01',
+    pncNumber = '01/2345A',
+    prisonNumber = 'G0015GF',
+    potentialMatches = [],
+  }) => ({
```

Comment:
helper method to allow creating arrivals in a way which emphasises the salient aspects required for the specific test